flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/reconciler/ReconciliationUtils.java

```diff
@@ -305,6 +308,7 @@ public static Duration rescheduleAfter(
             case READY:
                 rescheduleAfter =
                         savepointInProgress(flinkDeployment.getStatus().getJobStatus())
+                                        || isJobCancelling(flinkDeployment.getStatus())
```

Comment:
I think this will always be false based on line 276

Reply:
good catch